includes/class-smaily-for-wp-widget.php

```diff
@@ -17,6 +17,15 @@ class Smaily_For_WP_Widget extends WP_Widget {
 	 */
 	private $autoresponders;
 
+	/**
+	 * Handler for storing/retrieving data via Options API.
+	 *
+	 * @since  3.0.0
+	 * @access private
+	 * @var    Smaily_For_WP_Option_Handler $option_handler Handler for Options API.
+	 */
+	private $option_handler;
```

Comment:
You can also simply use `$options` here.

Reply:
I left the comments unchanged, they are still referencing an option handler.